inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/log/RecommenderEvaluationResultEventAdapter.java

```diff
@@ -66,7 +67,11 @@ public String getDetails(RecommenderEvaluationResultEvent aEvent)
             Details details = new Details();
 
             details.recommenderId = aEvent.getRecommender().getId();
-            details.score = aEvent.getScore();
+            EvaluationResult result = aEvent.getResult();
```

Comment:
how about including the training and test set sizes as well?

Reply:
good idea, i will do it